lib/componentRelay.ts

```diff
@@ -453,6 +453,8 @@ export default class ComponentRelay {
       link.className = 'custom-theme'
       this.contentWindow.document.getElementsByTagName('head')[0].appendChild(link)
     }
+
+    this.postMessage(ComponentAction.ThemesActivated, {})
```

Comment:
Do you think it would be better to reply to the original message instead?

Reply:
Do we need anything from the original message? If not, I think an empty object or `undefined` should be good.